docs/source/user-guide/integrations/h2o.rst

```diff
@@ -0,0 +1,157 @@
+============================
+H2O
+============================
+
+`H2O <https://docs.h2o.ai/h2o/latest-stable/h2o-docs/automl.html>`__ is
+a widely used AutoML framework, written in Java but with multiple fully
+implemented API libraries, including one in python. In this example,
+we’ll be showing how you can effortlessly test your H2O model with
+deepchecks in the exact same way you can test any sklearn model.
+
+This example can be run using
+`colab <https://colab.research.google.com/github/deepchecks/deepchecks/tree/examples/integrations/h2o/deepchecks_h2o_tutorial.ipynb>`__.
```

Comment:
Following our discussion. - we'll refer to our repo (with the py code) in the link ("note" at the top, like in the gallery notebooks)